### ISSUE TITLE
Need better documentation

### ISSUE THREAD

**Issue by swarupmohalik** [rubenvereecken/py-vgdl#9]
Need some good documentation for usage. In particular, how to add the details of a game (description, level etc) in __init__.py?

**Comment by rubenvereecken** [comment]
I agree, it could do with more documentation for different use cases. The best way would be a comprehensive small example of a RL agent.

I can get you started quickly however. If you check out [`play_vgdl.py`](https://github.com/rubenvereecken/py-vgdl/blob/master/vgdl/util/humanplay/play_vgdl.py), it takes a `levelfile` and `domainfile`. Ignoring all the rest of the parameters for now, it registers these with the VGDL registry, after which the environment can be accessed from anywhere through the registry. 

If I remember correctly, this basically sets up the VGDL interface, so you access VGDL through its own interface instead of Pybrain or Gym, which are other interfaces VGDL exposes if that's what you prefer.
